includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -184,7 +185,7 @@ public function render_course_outline_block( $attributes ) {
 			$block_class .= ' ' . $attributes['className'];
 		}
 
-		return '
+		return '	
```

Comment:
This whitespace will get stripped in my editor due to our `.editorconfig`. Was it added for a reason?

Reply:
Nope 😄 Not sure why PHPStorm doesn't respect it, I have the plugin installed. Probably has to do with my project structure 🤷